src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -57,6 +58,10 @@ public void RunApplication(Application app)
         {
             try
             {
+                //This line validates if the WebView2 Runtime is installed in the computer, if is not we return and then exit Dynamo
+                if (!WebView2Utilities.ValidateWebView2RuntimeInstalled())
+                    return;
```

Comment:
I think this is fine for now. For integrations, usually host will include WebView2 Runtime, for any integration launching Dynamo as another process, this should hit

Reply:
do we really need to exit Dynamo?

Reply:
@mjkkirschner It would be safe to do so since Splash Screen depending on WebView2, unless we want to totally skip that experience when runtime not installed